### PR TITLE
Automatic sign-out should redirect to the sign-in page

### PR DESCRIPTION
**Pull request by partpa**
#### What's this PR do? Any additional context?
     - when user clicks on signout it will redirect to signin page.

#### Where should the reviewer start?
     - signin into lms or cms and click signout .

#### How can this be manually tested? (brief repro steps)
     - Before : signout redirect to main home page.
       After : signout redirect to signin page.

#### What are the relevant TFS items? (list id numbers)
       Bug : 84811
 
#### Definition of done:
- [x ] Title of the pull request is clear and informative
- [ ] Add pull request hyperlink to relevant TFS items
- [ ] For large or complex change: schedule an in-person review session

[//]: # ( todo: Is there appropriate test coverage? )
[//]: # ( todo: Does this PR require a new Selenium test? )
[//]: # ( todo: Is there appropriate logging/monitoring included? )

#### Reminders BEFORE merging
1. Get at least two approvals
1. If you're merging into the development branch then "flatten" or "squash" commits
1. If merging from development into master then don't "flatten" or "squash" commits

#### Reminders AFTER merging
1. Delete the remote branch
1. Resolve relevant TFS items
1. (reverse merge) If you merged into master then check to see if there are any changes in master that can be merged down to the development branch (like hotfixes, etc)

[//]: # ( todo: If you merged into development branch then verify change in our "rolling deployment" environment. Then notify stakeholders interested in or involved with the change )

[//]: # ( fyi: This content was heavily inspired by )
[//]: # ( 1 Our team's policies and processes )
[//]: # ( 3 https://github.com/sprintly/sprint.ly-culture/blob/master/pr-template.md )
[//]: # ( 4 The book "The Checklist Manifesto: How to Get Things Right" by Atul Gawande )
[//]: # ( 5 https://github.com/Azure/azure-event-hubs/blob/master/.github/PULL_REQUEST_TEMPLATE.md )

### CODE REVIEW

**Comment by manikarthikk** [comment]
Is this a fix for LMS or CMS ? 
if LMS, then the url should be in LMS (lms/urls.py)

**Comment by partpa** [comment]
We have common logout for both LMS and CMS under common folder in views.py...but name of login page for LMS is login and for CMS is signup..so for LMS its working fine if i will only make change in views.py file. 
So only for CMS i have to add url.

**Comment by kabirkhan** [comment]
Please ensure that the tests still pass for this. The tests expect to redirect to the homepage url '/' not the '/login' url still.

Check:
'''
/common/djangoapps/student/tests/test_views.py in the LogoutTests class. 
'''

If you need help running the tests let me know.

File: common/djangoapps/student/views.py [hunk 1]
```diff
@@ -2607,7 +2607,7 @@ class LogoutView(TemplateView):
     template_name = 'logout.html'
 
     # Keep track of the page to which the user should ultimately be redirected.
-    target = reverse_lazy('cas-logout') if settings.FEATURES.get('AUTH_USE_CAS') else '/'
+    target = reverse_lazy('cas-logout') if settings.FEATURES.get('AUTH_USE_CAS') else '/login'
```
Comment (partpa) [inline common/djangoapps/student/views.py]: yes we can but  LMS and CMS has common function for signout redirection here.
so by default it is just '/' redirect to main Home page.
But Now we are redirecting it to singin page and the problem is...... in LMS page name is "login" and in CMS page name is "SIGNIN" so if will change here as /login it is causing error in CMS when i click on signout (As CMS has 'signin' page).

**Pull request merged**